src/App/Fossa/Config/Project/Edit.hs

```diff
@@ -130,11 +130,12 @@ projectIdentiferOptions = optional (projectIdOptions <|> projectLocatorOptions)
 
 mergeOpts ::
   (Has Diagnostics sig m) =>
+  Maybe FilePath ->
   Maybe ConfigFile ->
   EnvVars ->
   EditOpts ->
   m EditConfig
-mergeOpts maybeConfig envVars cliOpts@EditOpts{..} = do
+mergeOpts _ maybeConfig envVars cliOpts@EditOpts{..} = do
```

Comment:
I'm sticking the extra `_` in here and in other subcommands that do not have a debug flag so that we keep the signature of `mergeOpts` the same in all subcommands.